packages/ptokens-assets-evm/src/lib/index.ts

```diff
@@ -113,15 +109,14 @@ export const getOperationIdFromLog = (_log: Log, _networkId: NetworkId = null) =
   )
 }
 
-export const getOperationIdFromTransactionReceipt = (_networkId: NetworkId, _receipt: TransactionReceipt) => {
-  return getOperationIdFromLog(
-    _receipt.logs.find(
-      (_log) =>
-        _log.topics[0] === eventNameToSignatureMap.get(EVENT_NAMES.USER_OPERATION) ||
-        _log.topics[0] === eventNameToSignatureMap.get(EVENT_NAMES.OPERATION_QUEUED) ||
-        _log.topics[0] === eventNameToSignatureMap.get(EVENT_NAMES.OPERATION_EXECUTED) ||
-        _log.topics[0] === eventNameToSignatureMap.get(EVENT_NAMES.OPERATION_CANCELLED),
-    ),
-    _networkId,
+export const getOperationIdFromTransactionReceipt = (_networkId: NetworkId, _receipt: TransactionReceipt<bigint>) => {
+  const relevantLog = _receipt.logs.find(
+    (_log) => _log.topics[0] === eventNameToSignatureMap.get(EVENT_NAMES.USER_OPERATION), //||
+    // _log.topics[0] === eventNameToSignatureMap.get(EVENT_NAMES.OPERATION_QUEUED) ||
```

Comment:
No need to comment these. The function can get the correct operation id from the other logs as well.

Reply:
Inside the operationExecuted interim logs there are both operationExecuted and Usersend. If I add back all of the event types I have then to further filter them.